doc/man1/openssl-list.pod.in

```diff
@@ -10,6 +10,9 @@ openssl-list - list algorithms and features
 B<openssl list>
 [B<-help>]
 [B<-verbose>]
+{- output_off() if $disabled{"deprecated-3.0"}; ""
+-}[B<-backwards-compatibility>]
+{- output_on() if $disabled{"deprecated-3.0"}; ""
 [B<-select> I<name>]
```

Comment:
You forgot a closing brace

Reply:
I'll just undo this in favour of @beldmit's suggestion which seems better all around.